src/RoadRegistry.Wms.Projections/RoadSegmentRecordProjection.cs

```diff
@@ -0,0 +1,29 @@
+namespace RoadRegistry.Wms.Projections
+{
+    using System;
+    using System.Text;
+    using BackOffice.Messages;
+    using Be.Vlaanderen.Basisregisters.ProjectionHandling.Connector;
+    using Be.Vlaanderen.Basisregisters.ProjectionHandling.SqlStreamStore;
+    using Microsoft.IO;
+    using Schema;
+    using Schema.RoadSegmentDenorm;
+
+    public class RoadSegmentRecordProjection : ConnectedProjection<WmsContext>
+    {
+        public RoadSegmentRecordProjection(RecyclableMemoryStreamManager manager, Encoding encoding)
```

Comment:
You don't need RecyclableMemoryStreamManager nor Encoding (you're not dealing with shape files here).